src/printer.rs

```diff
@@ -119,6 +119,10 @@ pub struct Printer {
 }
 
 impl Printer {
+    pub fn set_stream(&mut self, stream: bool) {
+        self.stream = stream
+    }
+
```

Comment:
This will not be necessary if we check the content type inside `Printer::print_response_body`.